gateway/Cargo.toml

```diff
@@ -4,7 +4,7 @@
 [package]
 name = "nym-gateway"
 license = "GPL-3.0"
-version = "1.1.36"
+version = "1.1.37"
```

Comment:
I guess we just need to remember to not bump again during the release process

Reply:
I can leave it to 36 actually, initially I made the change because I was putting the update procedure in the gateway, but now that the PR is focused on nym-node, it can just stay to 36